tests/framework/Samplers/AdaptiveLimitSurfaceSearch/tests

```diff
@@ -11,4 +11,8 @@
   input = 'test_adaptive_sampler.xml'
   csv = 'Adapt/limitSurfaceDump.csv'
  [../]
+ [./adaptive_sampler_no_crossing]
+  type = 'RavenFramework'
+  input = 'test_limit_surface_no_crossing_transition.xml'
```

Comment:
No out check?

Reply:
Yes, because it does not generate any Limit Surface...the check is just to be sure we detect the error (i.e. we do not crash anymore)